burrito/cmd/clean.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```

Comment:
several of these with NAME HERE <EMAIL ADDRESS> that probably need to be addressed